Content.Server/Atmos/Monitor/Systems/AirAlarmSystem.cs

```diff
@@ -600,32 +593,17 @@ public void UpdateUI(EntityUid uid, AirAlarmComponent? alarm = null, DeviceNetwo
         var temperature = CalculateTemperatureAverage(alarm);
         var dataToSend = new Dictionary<string, IAtmosDeviceData>();
 
-        if (alarm.CurrentTab != AirAlarmTab.Settings)
+        foreach (var (addr, data) in alarm.VentData)
         {
```

Comment:
pretty sure whats happening is that the same device is in multiple tabs so its trying to add the same dict. value in twice. Might need to either break up dataToSend by tab or make the data not a single value but a list instead

Reply:
Didn't notice that failure, sorry. That's exactly what's happening. Made it a list, so it's easier to extend if another device comes along in the future.

Reply:
This is going to double-send data which seems wasteful, can't you just do containskey or something instead?

Reply:
Okay nevermind I checked it and they're big-ass separate classes.